### PR TITLE
Update cpp.md

### PR DESCRIPTION
**Pull request by nikcode9**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cpp/cpp.md
Comment (SSwiniarski) [inline content/cpp/cpp.md:13]:
```suggestion

There are many of versions of the C++ programming language. These versions of the language are implementations of compilers based on specifications constructed by the ISO C++ community, the community that oversees the development of the language. The following are versions of the language:
```
Needs a blank line between this and the preceding paragraph.

File: content/cpp/cpp.md
Comment (SSwiniarski) [inline content/cpp/cpp.md:19]:
```suggestion
- C++98 (ISO/IEC 14882:1998) is the first edition.
- C++03 (ISO/IEC 14882:2003) is the second edition.
- C++11 is the third edition.
- C++14 is the fourth edition.
- C++17 is the fifth edition.

```
Needs to be a bulleted list and have a blank line following it.

File: content/cpp/cpp.md
Comment (SSwiniarski) [inline content/cpp/cpp.md:20]:
```suggestion
These versions have many of differences, mainly additions to the standard library and expansion of APIs. These standards also have differences in APIs due to depreciation.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@nikcode9 Looks like a good addition. I did a first review and made some comments about the formatting and updating the language.

**Comment by SSwiniarski** [comment]
> @codecademy-docs please merge it.

@nikcode9 you need to make the changes above so I can approve it, then another maintainer has to review it. It's not going to be merged without any changes.

**Review by nikcode9 — COMMENTED** [review]
updated the file.

**Comment by nikcode9** [comment]
@SSwiniarski  updated it,please check.

**Review by SSwiniarski — APPROVED** [review]
@nikcode9 I did some minor edits for line spacing. Now it looks good for a second review.

**Review by nikcode9 — COMMENTED** [review]
updated

**Review by KTom101 — APPROVED** [review]
@nikcode9 This looks good as is.

**Pull request merged**
